src/lib.rs

```diff
@@ -32,27 +32,30 @@
 //! This becomes very useful when you have a lot of tests with verbose and
 //! potentially changing expected output.
 //!
-//! Under the hood, `expect!` macro uses `file!` and `line!` to record source
-//! position at compile time. At runtime, this position is used to patch the
-//! file in-place, if `UPDATE_EXPECT` is set.
+//! Under the hood, the `expect!` macro uses `file!`, `line!` and `column!` to
+//! record source position at compile time. At runtime, this position is used
+//! to patch the file in-place, if `UPDATE_EXPECT` is set.
 //!
 //! # Guide
 //!
 //! `expect!` returns an instance of `Expect` struct, which holds position
-//! information and a string literal. Use `Expect::assert_eq` for string
-//! comparison. Use `Expect::assert_debug_eq` for verbose debug comparison. Note
-//! that leading indentation is automatically removed.
+//! information and a string literal. Importing `expect!` under a different
+//! name is supported; `UPDATE_EXPECT` will continue to work.
+//!
+//! Use `Expect::assert_eq` for string comparison. Use
+//! `Expect::assert_debug_eq` for verbose debug comparison. Note that leading
+//! indentation is automatically removed.
 //!
 //! ```
-//! use expect_test::expect;
+//! use expect_test::expect as ex; // Note the renaming
```

Comment:
Let's *not* add this to the docs: this is a niche thing you shouldn't generally be doing. Documenting this helps for few people that do need this, but harms everyone else. Docs are like code -- the fewer lines you spend, the better!